enterprise/backend/test/metabase_enterprise/audit_app/api/collection_test.clj

```diff
@@ -21,6 +21,21 @@
                         :type))))
           (testing "GET /api/collection/test"
             (is (= "instance-analytics"
+                   (->> (mt/user-http-request :rasta :get 200 "collection/tree")
+                        last
+                        :type))))))))
+  (premium-features-test/with-premium-features #{}
+    (audit-db-test/with-audit-db-restoration
+      (audit-db/ensure-audit-db-installed!)
+      (t2.with-temp/with-temp [Collection _ {:name "Zippy"}]
+        (testing "Instance Analytics Collection should not show up when audit-app isn't enabled."
+          (testing "GET /api/collection"
+            (is (= nil
+                   (->> (mt/user-http-request :rasta :get 200 "collection")
+                        last
```

Comment:
It's guaranteed to be last when it does exist?

Reply:
Yea I think adam has a change that made it so